Refactoring of benchmarks (#133)
IntelPython/scikit-learn_bench: sklbench/report/README.md

# Report generator

> **Scikit-learn_bench** report generator creates high-level report with aggregated stats from provided benchmark results.

Reviewer: ```suggestion
**Scikit-learn_bench** report generator creates a high-level report with aggregated stats from provided benchmark results.
```

> Generator will support different types of report, but there is only one supported type currently:

Reviewer: ```suggestion
Generator will eventually support different types of reports, but there is only one supported type currently:
```

 - `Separate tables`: writes aggregated metrics on summary page and detailed result on separate pages

> Raw results are converted into pandas.DataFrame and final report is made by processing this dataframe into separate ones written into Excel tables.

Reviewer: ```suggestion
Raw results are converted into a pandas dataframe and the final report is made by processing this dataframe into Excel tables.
```

| `--split-columns`                              | str    | ['estimator', 'method', 'function']                 |                                                | Splitting columns for subreports/sheets.                                                                                         |
| `--diffs-selection`                            | str    | upper_triangle                                      | ['upper_triangle', 'lower_triangle', 'matrix'] | Selects which part of one-vs-one difference to show (all matrix or one of triangles).                                            |
| `--perf-color-scale`                           | float  | [0.8, 1.0, 10.0]                                    |                                                | Color scale for performance metric improvement in report.                                                                        |
| `--quality-color-scale`                        | float  | [0.99, 0.995, 1.01]                                 |                                                | Color scale for quality metric improvement in report.                                                                            |

---

> [Documentation tree](../../README.md#-documentation-tree)

Author: Fixed to correct header
